src/main/java/sx/blah/discord/handle/impl/obj/Message.java

```diff
@@ -82,14 +82,19 @@
 	protected volatile boolean isPinned;
 
 	/**
+	 * The NONCE token of the message
+	 */
+	protected volatile String nonce;
```

Comment:
Is it possible for the nonce to be updated via message edits? If so you need to handle when an edit event is received in the DiscordWS. Otherwise this should be `final` rather than `volatile`.